gm4_blast_furnaces/data/blast_furnaces/functions/create.mcfunction

```diff
@@ -0,0 +1,7 @@
+#@s - custom crafter with blast furnance recipe
+#called by recipe_check
+
+data merge entity @s {CustomName:'"gm4_blast_furnace"',ArmorItems:[{},{},{},{id:gray_stained_glass,Count:1}],Tags:["gm4_bf_output","gm4_no_edit","gm4_bf_has_furnace"],Silent:1}
```

Comment:
The hopper is called "Blast Furnace", the custom name is "gm4_blast_furnace", you get an advancement called "gm4:blast_furnace", but the tag is "gm4_bf_output"? 
I would like it to be just "gm4_blast_furnace" to be consistent with basically every other module. Changing this in the future would also break existing blast furnaces so I'd prefer this to be right from the start.

Reply:
As far as I'm aware (Kroppeb made this function) this was copied from Sparks

Reply:
My module only used one tag and that was gm4_blast_furnace